service/controller/chart/resource/release/create.go

```diff
@@ -100,7 +100,7 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		// The install will continue in the background. We set the checksum
 		// annotation so the update state calculation is accurate when we check
 		// in the next reconciliation loop.
```

Comment:
```suggestion
		// We set the hash annotation so the update state calculation is accurate
		// when we check in the next reconciliation loop.
```

Can we update the comment to match the name? 

The part about the installing continuing in the background is no longer accurate for Helm 3.

Reply:
You're right, this is only true when we have a tiller pod. Changing it now.